src/content/code/plugin-api/layers/findAll-layer.ts

```
@@ -0,0 +1,27 @@
+
+// Use the findAll method to get all visible layers
+const visibleLayers = reearth.layers.findAll(searchVisibleLayers);
+
+// Output the IDs of the found layers
+console.log(
+  "Visible layers found:",
+  visibleLayers.map((layer) => layer.id)
+);
+
+//2. Function to find all visible layers
+const visibleLayers = reearth.layers.findAll((layer) => layer.isVisible);
```

Biome

[error] Shouldn't redeclare 'visibleLayers'. Consider to delete it or rename it.

'visibleLayers' is defined here:


(lint/suspicious/noRedeclare)